pycaret/containers/models/time_series.py

```diff
@@ -910,6 +920,71 @@ def _set_tune_distributions(self) -> dict:
         return {}
 
 
+class ProphetContainer(TimeSeriesContainer):
+    def __init__(self, globals_dict: dict) -> None:
+        logger = get_logger()
+        np.random.seed(globals_dict["seed"])
+        self.gpu_imported = False
+
+        try:
+            from sktime.forecasting.fbprophet import Prophet  # type: ignore
+        except ImportError:
+            logger.warning("Couldn't import sktime.forecasting.fbprophet")
+            self.active = False
+            return
+
+        sp = globals_dict.get("seasonal_period")
+        self.sp = sp if sp is not None else 1
+
+        self.seasonality_present = globals_dict.get("seasonality_present")
+        self.freq = globals_dict.get("freq")
+
+        args = self._set_args
+        tune_args = self._set_tune_args
+        tune_grid = self._set_tune_grid
+        tune_distributions = self._set_tune_distributions
+
+        leftover_parameters_to_categorical_distributions(tune_grid, tune_distributions)
+
+        super().__init__(
+            id="prophet",
+            name="Prophet",
+            class_def=Prophet,
+            args=args,
+            tune_grid=tune_grid,
+            tune_distribution=tune_distributions,
+            tune_args=tune_args,
+            is_gpu_enabled=self.gpu_imported,
+            is_turbo=False,
+        )
+
+    @property
+    def _set_args(self) -> dict:
+        return {} 
+
+    @property
+    def _set_tune_args(self) -> dict:
+        return {}
+
+    @property
+    def _set_tune_grid(self) -> dict:
+        tune_grid = {
+            "seasonality_mode": ['additive', 'multiplicative'],
+            "growth": ['linear'] # 'logistic', cap column should be provided. https://stackoverflow.com/questions/48925255/fbprophet-giving-error-when-growth-logistic-python
+        }
+        return tune_grid
+
+    @property
+    def _set_tune_distributions(self) -> dict:
+        return {
+            # Based on https://facebook.github.io/prophet/docs/diagnostics.html#hyperparameter-tuning
+            # Gives a RunTimeError: Cannot clone object Prophet
+            #'changepoint_prior_scale': UniformDistribution(lower=0.01, upper=0.5, log=True),
+            #'seasonality_prior_scale': IntUniformDistribution(lower=1, upper=10),
+            #'holidays_prior_scale': UniformDistribution(lower=0.01, upper=10, log=True)
```

Comment:
```suggestion
            "seasonality_mode": CategoricalDistribution(['additive', 'multiplicative']),
            'changepoint_prior_scale': UniformDistribution(lower=0.001, upper=0.5, log=True),
            'seasonality_prior_scale': UniformDistribution(lower=0.01, upper=10, log=True),
            'holidays_prior_scale': UniformDistribution(lower=0.01, upper=10, log=True)
```
in accordance with facebook.github.io/prophet/docs/diagnostics.html#hyperparameter-tuning

Reply:
Fixed.